### ISSUE TITLE
Paste Location in Demo

### ISSUE THREAD

**Issue by CMTietgen** [layerssss/paste.js#52]
I was testing the demo in various browsers and got different results for where the image was pasted.

In the following browsers I clicked in the blank area **below** the 3rd div and the pasted image appeared **below** the 3rd div.

- Edge 38.14393.1066.0
- Firefox 54.0.1

In the following browsers I clicked **below** the 3rd div and the pasted image appeared **above** the 3rd div.

- Chrome 59.0.3071.115

For IE11 11.1480.14393.0 the image wouldn't past unless I clicked into one of the 3 divs on screen.

For all browsers, clicking inside one for the 3 sample divs, the image is pasted below the selected div.

I would image the different paste locations when clicking below the 3rd div is just a quirk in the demo, right?

**Comment by layerssss** [comment]
@CMTietgen True, that's a quirk, (not sure in the library or just in the demo), since you shouldn't be able to paste if you click `below` the div. I'll look into it.
